### PR TITLE
feat: swap fox farming v4 for v5

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

V5 contract starting November 18th 9AM MT, this will prepare us for it and should automagically work once it's live

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

Ensure current opportunities are still visible, see testing steps

## Testing

- Refresh with a fresh account or an account without FOX farming staking
- Ensure you're still seeing V4 as a farming opportunity in DeFi earn

- Refresh with an account with V4 staking
- Ensure you're still seeing your opportunity in DeFi earn/overview

### Engineering

- ☝🏽 for regression testing

To test the actual V5 feature:

- Apply this diff:

```shell
diff --git a/src/components/StakingVaults/AllEarnOpportunities.tsx b/src/components/StakingVaults/AllEarnOpportunities.tsx
index 6b6270370..d4e7213cb 100644
--- a/src/components/StakingVaults/AllEarnOpportunities.tsx
+++ b/src/components/StakingVaults/AllEarnOpportunities.tsx
@@ -70,7 +70,7 @@ export const AllEarnOpportunities = () => {
   })
 
   const filteredRows = useMemo(() => {
-    return allRows.filter(e => bnOrZero(e.tvl).gte(50000))
+    return allRows
   }, [allRows])
 
   const handleClick = useCallback(
```
- Ensure you're seeing this (we currently filter out opportunities by TVL, so we won't be seeing it until it goes live):

image

### Operations

- ☝🏽 
## Screenshots (if applicable)

image
image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3278** <a href="https://app.graphite.dev/github/pr/shapeshift/web/3278" target="_blank">Graphite</a> 
    * **PR #3280** <a href="https://app.graphite.dev/github/pr/shapeshift/web/3280" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/3280?utm_source=stack-comment).

**Comment by gomesalexandre** [comment]
Opening to make sure we get this one when the new contract goes live, will fix conflicts after https://github.com/shapeshift/web/pull/3218 goes in

**Comment by 0xApotheosis** [comment]
Converting to draft as this has build errors and merge conflicts.

Re-open once we are ready to go 👌

**Review by 0xApotheosis — COMMENTED** [review]
Conceptually this looks fine - will need to re-test once contract is live ✅

File: src/features/defi/providers/fox-farming/api.ts [hunk 1]
```diff
@@ -1,99 +0,0 @@
-import { Contract } from '@ethersproject/contracts'
```
Comment (gomesalexandre) [inline src/features/defi/providers/fox-farming/api.ts:1]: Was actually unused after the refactor

**Review by 0xApotheosis — APPROVED** [review]

**Pull request merged**